### ISSUE TITLE
Ubuntu versions of tutorials

### ISSUE THREAD

**Issue by Rick-Anderson** [aspnet/Docs#1799]
[customer asks](https://docs.asp.net/en/latest/getting-started.html#comment-2845326142):
I know Linux is less important initially than Windows or Mac, but would love to see more documentation focused on the Linux side of things, like taking the first ASP.NET core app on a Mac using Visual Studio Code and making a Linux (including Ubuntu, preferably) version of it. I'm primarily a Windows guy who is dabbling in Ubuntu currently. What an exciting technology.

dup of #1215 and #2993

**Comment by Iarflaith** [comment]
Hi, I followed the tutorial where this comment was made, using Ubuntu 16.04.

At first I had two errors. Before debugging: In Program.cs, error CS0246 "The type or namespace name 'Startup' could not be found (are you missing a using directive or an assembly reference?) aspNetCoreApp.Startup"

And after debugging: "launch: program '/home/path/aspNetCoreApp/bin/Debug/netcoreapp1.0/aspNetCoreApp.dll' does not exist"

I was worried that Startup.cs was in the wrong place, and that I messed up using camelCase for my namespaces, but I ran it again and it worked fine. 

I'm not experienced enough to say why the dll file wasn't there the first time, and I also originally named my file startup.cs instead of Startup.cs, but in any case the tutorial currently works for Ubuntu and VSCode users.
